tpythia6.sh

```diff
@@ -0,0 +1,40 @@
+package: TPythia6
+version: "main"
+tag: "main"
+source: https://github.com/ShipSoft/TPythia6
+requires:
+  - ROOT
+  - pythia6
+build_requires:
+  - CMake
+prepend_path:
+  ROOT_INCLUDE_PATH: "$TPYTHIA6_ROOT/inc"
+---
+#!/bin/bash -e
+cmake "$SOURCEDIR"                               \
+    -DPYTHIA6_ROOT=${PYTHIA6_ROOT} \
+
+make 
+cp -r $SOURCEDIR/inc $INSTALLROOT/inc
+cp *.so $INSTALLROOT/
```

Comment:
`Use ./*glob* or -- *glob* so names with dashes won't become options. [SC2035]`

Maybe all these copies should by handled by CMake's install step?